src/components/nodes/signalEndEvent/signalEndEvent.vue

```diff
@@ -0,0 +1,21 @@
+<script>
+import EndEvent from '@/components/nodes/endEvent/endEvent';
+import signalEndEventIcon from '@/assets/signal-end-event.svg';
+import pull from 'lodash/pull';
+
+export default {
+  extends: EndEvent,
+  props: ['rootElements'],
+  mounted() {
+    this.shape.attr({
+      image: {
+        'ref-y': 2,
+        'xlink:href': signalEndEventIcon,
+      },
+    });
+  },
+  destroyed() {
+    pull(this.rootElements, this.error);
+  },
```

Comment:
Might have missed this when pairing with you–sorry. But since we're not creating an error element, we don't have to worry about removing it. So this whole `destroyed` block (along with the single prop it was accepting) can be removed.

Reply:
Super, will do